Beginner Class Updates (#261)
mitre/saf-training: src/courses/beginner/07.md

Well. . . it *sort of* works.

Notice how much output InSpec printed here to answer the simple question of "did this command return empty?" Imagine if we had done this on a directory with many files in it. We'd just be cluttering up the screen (and our report files) with output that is only tangential to what we really want to know, which is that the directory is _not empty_; we don't care what's in it.

::: warning Wait, couldn't we have just used the directory resource for this?
Correct. That would have been a much better way of doing this, and illustrates the general principle of "don't use raw shell commands with the `command` resource unless you have to."

We're just doing it this way for the example.
:::

> If we would like to have a more detailed and cleaner feedback to our user, we can override the standard title of our describe block with a specific message that describes the intent of the test and use the [Explicit Subject](https://rspec.info/features/3-12/rspec-core/subject/explicit-subject/) to tell InSpec what the "subject" is for the test, then, we could refactor the code like this:

Reviewer: ```suggestion
If we would like to have a more detailed and cleaner feedback to our user, we can override the standard title of our describe block with a specific message that describes the intent of the test and use the [`subject` keyword](https://rspec.info/features/3-12/rspec-core/subject/explicit-subject/) to tell InSpec what the "subject" is for the test.  This allows us to refactor the code like this:
```
